src/onegov/town6/templates/macros.pt

```diff
@@ -834,6 +834,16 @@
                     <li tal:condition="person.website and 'website' not in exclude" class="person-card-website">
                         <a href="${person.website}">${person.website}</a>
                     </li>
+                    <div class="person-card-context-specific-functions" tal:condition="python: len(functions) != 0 and len(organizations) != 0 and organizations is not None and functions is not None">
+                        <div tal:define="map python: zip(organizations, functions)">
```

Comment:
If you intend to use it like this, you should probably pass it at a list of lists  in the first place (which also less error prone). Why not pass only only one string with "organization: function" already anyway?

Reply:
I can do this, but why is it error prone? When using  `map` wouldn't the iterator stop when the shortest input iterable is exhausted, e.g. the if it somehow happens the lists are not equal in length.